pkg/drivers/krunkit/krunkit.go

```diff
@@ -286,14 +286,20 @@ func (d *Driver) setupIP(mac string) error {
 		return nil
 	}
 	// Implement a retry loop because IP address isn't added to dhcp leases file immediately
```

Comment:
Add a constant for maxAttempts since we need it twice:

```go
const maxAttempts = 60
```

This will also make it more consistent with other drivers, which will make it easier to extract this loop to a helper function, since all the drivers use the similar logic.